### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by GuptaEesh**
Buttons contain:
**Differentiations**
> Simple Buttons
  1. Primary
  2. Secondary
  3. Tertiary
> Link-Button
   1. Primary-Link
   2. Secondary-Link
   3. Image-Link
> Icon-Buttons
   1. Image-Icons
   2. Material-Icons
> Reveal Buttons
   
buttons

### CODE REVIEW

File: components/buttons.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
+.btn {
+    border: 0;
+    border-radius: 0.25rem;
+    min-width: 1rem;
+    padding: 3.2px 2rem;
+
+}
+
+.btn:hover {
```
Comment (shivampandey0) [inline components/buttons.css:9]: IMO this is not needed, `cursor: pointer` can be added in the above `btn` class.

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions

**Review by ZuberDunge — COMMENTED** [review]
looks good to me

File: components/buttons.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
+.btn {
+    border: 0;
+    border-radius: 0.25rem;
+    min-width: 1rem;
+    padding: 3.2px 2rem;
```
[the remark this answers is not in the document]
Comment (GuptaEesh) [reply components/buttons.css:5]: Yup, I will look into it.

File: components/buttons.css (hunk 1 above)
Comment (GuptaEesh) [reply components/buttons.css:9]: It's just for adding readability and future-readiness if any changes are made.

**Pull request merged**